src/configuration/js-team/appDefaults.js

```diff
@@ -0,0 +1,8 @@
+const DEFAULT_DESKTOP_BENCHMARKS = {
+  landingPath: '/win10/overview?numDays=90',
+  timeRange: 90, // 90 days
+  colors: ['#e55525', '#ffcd02'],
+  labels: ['Firefox', 'Chrome'],
+};
+
+export default DEFAULT_DESKTOP_BENCHMARKS;
```

Comment:
Call it DASHBOARD_CONFIG since it is not specific to the desktop browser.
```suggestion
export default DASHBOARD_CONFIG;
```